admin/config/environments/staging/request.json

```diff
@@ -0,0 +1,23 @@
+{
+  "session": {
+    "enabled": true,
+    "client": "cookie",
+    "key": "strapi.sid",
+    "prefix": "strapi:sess:",
+    "secretKeys": ["mySecretKey1", "mySecretKey2"],
+    "httpOnly": true,
+    "maxAge": 86400000,
```

Comment:
we should look into making this infinity -> otherwise our API user may have to log back in again